src/main/java/winstone/Launcher.java

```diff
@@ -183,7 +182,7 @@ public Launcher(Map<String, String> args) throws IOException {
             }
 
             // JENKINS-73616: Turn down log level of annotation parser
-            java.util.logging.Logger logger = java.util.logging.Logger.getLogger(AnnotationParser.class.getName());
+            java.util.logging.Logger logger = java.util.logging.Logger.getLogger("org.eclipse.jetty.ee9.annotations.AnnotationParser");
```

Comment:
This annotation is no longer on our classpath in EE 9 due to some upstream cleanup in the Jetty dependency tree, so just use a string constant.